src/libcommonserver/io/iohelper.h

```diff
@@ -173,7 +173,7 @@ struct IoHelper {
           \param ioError holds the error associated to a failure of the underlying OS API call, if any.
           \return true if no unexpected error occurred, false otherwise.
         */
-        static bool createSymlink(const SyncPath &targetPath, const SyncPath &path, IoError &ioError) noexcept;
+        static bool createSymlink(const SyncPath &targetPath, const SyncPath &path, bool isFolder, IoError &ioError) noexcept;
```

Comment:
The docstring above needs to be adapted.

Reply:
Done